### DISCUSSION TITLE
Early Game Balance

### DISCUSSION THREAD

**Discussion by ultidonki** [Tuxemon/Tuxemon#2313] [General]
If you've started playing the game from the beginning, discuss things you've found frustrating or annoying here. Here's some stuff I've found:

- <s>Technique: Wallow is overpowered. Eg. Hibernate heals you, but puts tuxemon to sleep for 3 turns. Wallow also heals you, but has NO cooldown and NO bad side effects.</s>
- Run away - I tried to run away more than 10 times from a level 8 cataspike, my tuxemon were only level 3-6, none of them could escape so it slowly wiped out my entire team of 6.
- Levelling up in the wild: EXP from trainers gives you a huge amount. EXP from random battles gives you almost nothing. Eg. a level 10 tuxemon defeating a level 8 tuxemon only got 8 EXP (They need more than 100 to level up). I don't think I've ever got a level up from battling in the wild, only during trainer battles.
- First rival battle: After fighting level 2-5 tuxemon on route 1, the rival appears with 2 level 6 tuxemon and 1 level 3 tuxemon. I struggled a lot with this battle, (see how little experience you get fighting tuxemon in the wild), but eventually won after I had 4 tuxemon in my party, even if they were only level 2. So I feel this is only winnable if you have more tuxemon than them. Which causes the next problem:
- 'Do not consume items on failure' should be turned on by default. In the early game, it's very hard to earn money, so wasting all your money on tuxeballs which fail to catch a level 2 cataspike sucks. (Especially as you NEED to catch several tuxemon to defeat the 2nd rival battle) (or any of the other trainers that follow).
- <s>There should be a cathedral nurse in the 2nd part of route 2 (the city park). Otherwise, if your tuxemon are tired, the only option is: 1) walk back 2 maps to the previous city to get healed, or 2) die and get warped back to the previous city. I'd place the nurse right under the red-roofed building near the bottom of the map.</s>

I will say something positive: It's a lot more balanced and fun than it used to be. Particularly all the status effects, and type match ups make me consider changing my tuxemon a lot more often during a battle, whereas in my first playthrough a few years ago I just had 1 overpowered rockitten that steamrolled everything. :smile:

**Comment by Davidliang11292** [comment]
During combat, some prompts lingered for a long time, which I think caused the tests to drag on pointlessly (but this could also be because I was testing in Chinese), while some prompts were almost as short as a flash, which I thought was bad for translation (by the way, I'm going to overhaul the Chinese translation of NPC dialogue, Because some conversations don't seem to flow)

**Reply by ultidonki** [reply]
Interesting - the prompts seemed OK to me in the English version, but maybe I'll test in the Chinese version too and see if the prompts are different there.

**Reply by Davidliang11292** [reply]
I've observed that many of these prompts are related to animations, and what requires several words in English can often be conveyed in just a few characters in Chinese. Additionally, Chinese features "chengyu" – four-character idioms that express complex ideas succinctly; an example being "同心协力(Pinyin: tóng xīn xié lì)" which depicts a group of people with shared aspirations working together towards a common goal.

Furthermore, when you switch Tuxemon's language to Chinese, you'll notice that most dialogues don't exceed a single line in the dialogue box.

Getting back to the point, this results in a situation where, in the Chinese version of Tuxemon, the prompts conclude rather quickly, sometimes just as the animation has barely started for about two seconds, whereas in the English version, the timing of the prompt ending and the animation finishing is much closer together.

This leads to instances where prompts linger unnecessarily or vanish prematurely, creating inconsistencies in the user experience under certain circumstances.

**Reply by ultidonki** [reply]
Thank you for the detailed reply! I don't know how to fix it, but I understand the problem now. Maybe if you make a new issue for it, people will be able to see the problem and track it better. :smiley:

**Reply by Davidliang11292** [reply]
> Thank you for the detailed reply! I don't know how to fix it, but I understand the problem now. Maybe if you make a new issue for it, people will be able to see the problem and track it better. :smiley:

You're welcome, but I don't know how to fix it either, the only solution I can think of is to make the Chinese prompts longer

**Comment by Sanglorian** [comment]
Thanks for sharing your thoughts! Very useful. 

> If you've started playing the game from the beginning, discuss things you've found frustrating or annoying here. Here's some stuff I've found:
> 
>     * Technique: Wallow is overpowered. Eg. Hibernate heals you, but puts tuxemon to sleep for 3 turns. Wallow also heals you, but has NO cooldown and NO bad side effects.

Good point - on the wiki I've changed it to heal half as much, have a one-round cooldown and a 10% chance of not working. 

>     * Run away - I tried to run away more than 10 times from a level 8 cataspike, my tuxemon were only level 3-6, none of them could escape so it slowly wiped out my entire team of 6.

Does someone know what the current formula is for this? I'm sure we could tweak it to make it fairer. (I'm not sure if it ever made it onto the wiki!)

>     * Levelling up in the wild: EXP from trainers gives you a huge amount. EXP from random battles gives you almost nothing. Eg. a level 10 tuxemon defeating a level 8 tuxemon only got 8 EXP (They need more than 100 to level up). I don't think I've ever got a level up from battling in the wild, only during trainer battles.

By my calculations from the formula described on the [wiki](https://wiki.tuxemon.org/Experience), a level 10 tuxemon should need 100 XP to level up and should get 64 from a level 8 tuxemon, rather than 8. Perhaps the formula has been programmed as "level" when it should be "level ^2"?  

>     * First rival battle: After fighting level 2-5 tuxemon on route 1, the rival appears with 2 level 6 tuxemon and 1 level 3 tuxemon. I struggled a lot with this battle, (see how little experience you get fighting tuxemon in the wild), but eventually won after I had 4 tuxemon in my party, even if they were only level 2. So I feel this is only winnable if you have more tuxemon than them. Which causes the next problem:

I think this should be fixed if we use a more generous XP formula. 
 
>     * 'Do not consume items on failure' should be turned on by default. In the early game, it's very hard to earn money, so wasting all your money on tuxeballs which fail to catch a level 2 cataspike sucks. (Especially as you NEED to catch several tuxemon to defeat the 2nd rival battle) (or any of the other trainers that follow).

I like the economics of taking a chance on throwing a tuxeball. But it does look like the catch formula needs to be made much more generous! Does anyone know if it's currently using the formula described [on the wiki](https://wiki.tuxemon.org/Capturing)? If I understand it correctly, I would suggest multiplying that number by 255! That would give odds of 71% of a capture when a 100% catch rate monster is at 1/4 HP (if my maths is correct).

My apologies - I think I introduced that flaw in the maths when I put the formula together.  

>     * There should be a cathedral nurse in the 2nd part of route 2 (the city park). Otherwise, if your tuxemon are tired, the only option is: 1) walk back 2 maps to the previous city to get healed, or 2) die and get warped back to the previous city. I'd place the nurse right under the red-roofed building near the bottom of the map.

Sounds good to me. 

> I will say something positive: It's a lot more balanced and fun than it used to be. Particularly all the status effects, and type match ups make me consider changing my tuxemon a lot more often during a battle, whereas in my first playthrough a few years ago I just had 1 overpowered rockitten that steamrolled everything. 😄

**Comment by JaskRendix** [comment]
> 'Do not consume items on failure' should be turned on by default. In the early game, it's very hard to earn money, so wasting all your money on tuxeballs which fail to catch a level 2 cataspike sucks. (Especially as you NEED to catch several tuxemon to defeat the 2nd rival battle) (or any of the other trainers that follow).

@ultidonki  I pulled some data about the 2nd rival and the real issue is **eyenemy**, since the first choice and the bird aren't an issue, this one is overpowered because of [ruby](https://wiki.tuxemon.org/Ruby) , I think if we switch ruby with [strike](https://wiki.tuxemon.org/Strike) or [bullet](https://wiki.tuxemon.org/Bullet), then it could be less tough (@Sanglorian )

> Technique: Wallow is overpowered. Eg. Hibernate heals you, but puts tuxemon to sleep for 3 turns. Wallow also heals you, but has NO cooldown and NO bad side effects.

and 

> There should be a cathedral nurse in the 2nd part of route 2 (the city park). Otherwise, if your tuxemon are tired, the only option is: 1) walk back 2 maps to the previous city to get healed, or 2) die and get warped back to the previous city. I'd place the nurse right under the red-roofed building near the bottom of the map.


Both fixed in #2317 

> Levelling up in the wild: EXP from trainers gives you a huge amount. EXP from random battles gives you almost nothing. Eg. a level 10 tuxemon defeating a level 8 tuxemon only got 8 EXP (They need more than 100 to level up). I don't think I've ever got a level up from battling in the wild, only during trainer battles.

That because of the **exp_req_mod** (eg db/encounter/route1), so pairagrin lv 5 > 5 exp:
```
  "monsters": [
    {
      "monster": "pairagrin",
      "encounter_rate": 3.5,
      "variable": "daytime:true",
      "exp_req_mod": 1,
      "level_range": [
        2,
        4
      ]
    },
```
we can mass modify without any issue, @Sanglorian needs to pick a new multiplier for wild monsters: 3? 5?

> Run away - I tried to run away more than 10 times from a level 8 cataspike, my tuxemon were only level 3-6, none of them could escape so it slowly wiped out my entire team of 6.

@Sanglorian here:
```
            escape = 0.4 + (0.15 * (attempts successful + level_user - level_target))
            if random.random() <= escape:
                escaping = True
```
so in your case, lv 5 vs lv 8 cataspike (remember random.random() goes from 0 to 1):
```
0 successful attempts: random.random() <= -0.05 (so no escape at all)
1 successful attempts: random.random() <= 0.1
2 successful attempts: random.random() <= 0.25
3 successful attempts: random.random() <= 0.4
4 successful attempts: random.random() <= 0.55
5 successful attempts: random.random() <= 0.7
[...]
7 successful attempts: random.random() <= 1 (from here, it'll result in 100% escape)
```
it depends where do you want to draw the line.

totally random, lv 42 vs lv 69 (wild), some data:
```
0	attemps	-3.65 always failing
20	attemps	-0.65 always failing
25	attemps	0.1
30	attemps	0.85
31	attemps	1 always escaping
```

**Reply by Sanglorian** [reply]
Thanks @JaskRendix - good to have your expertise in this discussion. 

> @ultidonki I pulled some data about the 2nd rival and the real issue is **eyenemy**, since the first choice and the bird aren't an issue, this one is overpowered because of [ruby](https://wiki.tuxemon.org/Ruby) , I think if we switch ruby with [strike](https://wiki.tuxemon.org/Strike) or [bullet](https://wiki.tuxemon.org/Bullet), then it could be less tough (@Sanglorian )

I think we swap Ruby for Strike.

> > Levelling up in the wild: EXP from trainers gives you a huge amount. EXP from random battles gives you almost nothing. Eg. a level 10 tuxemon defeating a level 8 tuxemon only got 8 EXP (They need more than 100 to level up). I don't think I've ever got a level up from battling in the wild, only during trainer battles.
> 
> That because of the **exp_req_mod** (eg db/encounter/route1), so pairagrin lv 5 > 5 exp:
> 
> ```
>   "monsters": [
>     {
>       "monster": "pairagrin",
>       "encounter_rate": 3.5,
>       "variable": "daytime:true",
>       "exp_req_mod": 1,
>       "level_range": [
>         2,
>         4
>       ]
>     },
> ```
> 
> we can mass modify without any issue, @Sanglorian needs to pick a new multiplier for wild monsters: 3? 5?

I think it's that all XP should be calculated as (level^2)*exp_req_mod whereas if I understand correctly it's currently (level)*exp_req_mod? 

So it's okay for wild monsters to have exp_req_mod = 1, but all XP regardless of source should start as level ^2 (if my maths is correct, which it very well may not be!)

> @Sanglorian here:
> 
> ```
>             escape = 0.4 + (0.15 * (attempts successful + level_user - level_target))
>             if random.random() <= escape:
>                 escaping = True
> ```
> 
> so in your case, lv 5 vs lv 8 cataspike (remember random.random() goes from 0 to 1):
> 
> ```
> 0 successful attempts: random.random() <= -0.05 (so no escape at all)
> 1 successful attempts: random.random() <= 0.1
> 2 successful attempts: random.random() <= 0.25
> 3 successful attempts: random.random() <= 0.4
> 4 successful attempts: random.random() <= 0.55
> 5 successful attempts: random.random() <= 0.7
> [...]
> 7 successful attempts: random.random() <= 1 (from here, it'll result in 100% escape)
> ```
> 
> it depends where do you want to draw the line.
> 
> totally random, lv 42 vs lv 69 (wild), some data:
> 
> ```
> 0	attemps	-3.65 always failing
> 20	attemps	-0.65 always failing
> 25	attemps	0.1
> 30	attemps	0.85
> 31	attemps	1 always escaping
> ```

Thanks, it's super useful to see it laid out like that.

What if the formula were instead:

```
            escape = ((0.4 + (0.15 * attempts successful)) * (level_user / level_target))
            if random.random() <= escape:
                escaping = True
```

Then level 5 vs level 8 works out at:

Attempt 1: 34%
Attempt 2: 44%
Attempt 3: 53%
Attempt 4: 63%
Attempt 5: 72% 
etc

More achievable in the early attempts but slower to increase in achievability with more attempts. Interested to know what you think @ultidonki and @JaskRendix

**Reply by ultidonki** [reply]
@ultidonki I pulled some data about the 2nd rival and the real issue is eyenemy, since the first choice and the bird aren't an issue, this one is overpowered because of [ruby](https://wiki.tuxemon.org/Ruby) , I think if we switch ruby with [strike](https://wiki.tuxemon.org/Strike) or [bullet](https://wiki.tuxemon.org/Bullet), then it could be less tough 

It was a while since I fought this battle, sorry. I can't remember what the issue was. I think the enemies just took too long to defeat, so they could get an equal number of hits on my tuxemon - therefore, if they have 3 tuxemon, if my tuxemon are the same level, we have an equal chance of winning. If I only have 1 or 2, I'll probably get defeated, and if I have 4 or more, I'll probably win. 

If there was an opportunity to catch more monsters somewhere, or level up faster, it might be easier. For example, if I catch 2 tuxemon in the first route, they'll probably only be level 2 or 3, so we need to battle a lot to get any experience (and because Billie blocks the way, the only place I can level up is by fighting other level 2 or 3 tuxemon in the first route). 

That change is probably fine if you think the 3rd tuxemon is too hard to defeat. When I have time I can try it again with the 3rd one weaker, and see if that makes it any easier. :smile: 

Re: Running away:
> More achievable in the early attempts but slower to increase in achievability with more attempts.

I don't know how the numbers will 'feel' when playing the game, if it will feel too easy or hard to run away, but it looks much better than previously. Having to 'run away' 25 times before you have a 10% chance of escaping seems crazy. If I'm running away it's because I think I won't survive the next 5 or so turns. I'd prefer to attack 25 times if I knew it would take that long :rofl:

**Reply by Sanglorian** [reply]
Thanks @ultidonki , that all makes sense. I've swapped Ruby for Bullet in Eyenemy's wiki page. 

Leveling up should get easier if I can track down how the XP is being calculated - I think it might be less than it should be:

> I think it's that all XP should be calculated as (level^2)*exp_req_mod whereas if I understand correctly it's currently (level)*exp_req_mod?
> So it's okay for wild monsters to have exp_req_mod = 1, but all XP regardless of source should start as level ^2 (if my maths is correct, which it very well may not be!)

Other creature catching games give players quite a few boosts, which we haven't done in Tuxemon. For example, the starter Pokemon are more powerful than other Pokemon of the same level. A level 5 Bulbasaur has higher average stats than a level 5 Caterpie, for example. 

It also gives trained Pokemon extra stat "experience" that it doesn't give wild Pokemon - that's another reason why a level 8 or 9 Bulbasaur can fight through so many level 3 or 4 Weedles - it will have higher stats thanks to that different type of "Experience". 

I don't know what we should do about this for Tuxemon. One option would be to try to include something like this in Tuxemon - different stats for different mons, and higher stats for starters, for example. Another option would be to just give the player a higher-level starter mon. That would make the early levels easier.

**Reply by JaskRendix** [reply]
@Sanglorian
```
exp_tot = (loser.level + LEVEL_OFS) ** 3
exp_tot = (5 + 0) ** 3
exp_tot = 5 ** 3
exp_tot = 125
```
1. `result = (exp_tot // (loser.level * hits)) * exp_mod` where `exp_tot = (loser.level + level_ofs) ** 3`
```
result = (exp_tot // (loser.level * hits)) * EXP_MOD_1
result = (125 // (5 * 1)) * 1
result = (125 // 5) * 1
result = 25 * 1
result = 25
```

2. `result = (exp_tot // ((loser.level ** 2) * hits)) * exp_mod` ? (** 2 = 5**5 = 25)
```
result = (exp_tot // ((loser.level ** 2) * hits)) * EXP_MOD_1
result = (125 // ((5 ** 2) * 1)) * 1
result = (125 // (25 * 1)) * 1
result = (125 // 25) * 1
result = 5 * 1
result = 5
```
![Figure_1](https://github.com/user-attachments/assets/eb75af49-3abc-4256-ab6b-6c584303f18a)

**Reply by JaskRendix** [reply]
You asked me also about escaping formula:

old:
```
            escape = 0.4 + (0.15 * (attempts successful + level_user - level_target))
            if random.random() <= escape:
                escaping = True
```
new:
```
            escape = ((0.4 + (0.15 * attempts successful)) * (level_user / level_target))
            if random.random() <= escape:
                escaping = True
```

![Figure_1](https://github.com/user-attachments/assets/636c372a-9a40-47b9-bff3-fd65c8cb5ef0)
![Figure_2](https://github.com/user-attachments/assets/3f07a26d-114a-4c2d-ab58-6a21eefea8f8)
![Figure_3](https://github.com/user-attachments/assets/30836a87-a1db-49ca-b6df-04a5f12ee9a1)

you see, it gets better, but I was thinking that maybe a new multiplier can help (#2331), let me know if you want to improve it.

**Reply by HippasusTwo** [reply]
I'm going to be honest, I'd kind of like it if escape attempts for the first route were always successful unless you've activated some kind of 'hard mode'. I mean... It's the first route. But this improvement does seem alright to me.

I think there should be a campaign start settable variable called 'ease of running' which acts as a multiplier. Something like 1 = normal difficulty, 0 = impossible to run difficulty, 0.5 = fairly difficult but not impossible, 100 = super easy. Or perhaps it should be 0 = normal/easy difficulty, in case someone forgets to set the var for their campaign.

**Reply by ultidonki** [reply]
@JaskRendix  
Are the results in your 'Experience Points' graph based on the current formula, or a new formula? 

It seems a bit strange that my level 12 tuxemon (which can beat most tuxemon in 1 or 2 hits) will get more experience points than my level 5 tuxemon (which may take 7-10 hits to win against the same tuxemon) but I guess it's fine - I'll get more points fighting battles with other weaker tuxemon, so my level 5 tuxemon can win with less hits.
So that's fine to me.

However, in the graph, it looks like the experience points goes down to 0 if you take too long to defeat the tuxemon? Is that correct, or does 0 mean '1' here, like, multiply by 1? It's a bit harsh if I beat a reeeeeally hard tuxemon in a battle that took lots of hits, and then only get 1 or 0 EXP points, or will there be a lower cap eg. if it's level 50, I'll always get at least 50 EXP points? (Although even for a level 10 tuxemon, it takes ~100 EXP to level up... so 50 EXP isn't much in this case...)

1 other question, sorry - Do these hits roll over if the tuxemon changes? For example, if my level 15 tuxemon uses 10 hits to get a tuxemon down to 1 HP, and then I switch to a level 3 tuxemon (or, that level 15 tuxemon is defeated, and I switch then), and defeat it with 1 attack - will that level 3 tuxemon get a HUGE experience points boost, as it only took 1 hit, or do all EXP roll over so it took 11 hits? 

Overall, I think as long as I get a lot more EXP for wild battles, I'll be happier, I don't entirely understand the maths but currently needing to defeat 10-20 wild tuxemon to get 1 level higher is not very fun. Any multiplier at the minute would make me happier :smiley:

**Reply by JaskRendix** [reply]
@ultidonki 

>  Are the results in your 'Experience Points' graph based on the current formula, or a new formula?

Blue = actual formula
Red = actual formula with @Sanglorian correction

> 1 other question, sorry - Do these hits roll over if the tuxemon changes? For example, if my level 15 tuxemon uses 10 hits to get a tuxemon down to 1 HP, and then I switch to a level 3 tuxemon (or, that level 15 tuxemon is defeated, and I switch then), and defeat it with 1 attack - will that level 3 tuxemon get a HUGE experience points boost, as it only took 1 hit, or do all EXP roll over so it took 11 hits?

No, the formula doesn't redistribute exp based on how many X or Y monster hit. If total hits are 10, X hit 1/10 and X hit 9/10, then it doesn't consider it. 

Look, there are already two additional method for calculating experience that I added back then, do this, open the tmx file, and add this line before starting the battle:
`    <property name="act69" value="set_variable method_experience:???"/>`
replace **act69** with the correct number and replace **???** with:
- proportional (this is proportional, it takes in consideration how many times a monster hit, so it replies to your question)
- test (this is my creation, it uses different parameters and it doesn't consider hits)
- xp_transmitter (same formula but redistribute equally the exp points, as per item)

let me know

**Reply by ultidonki** [reply]
Thank you for replying so quickly, JaskRendix :smile:  

I'll test these other experience methods as I keep playing, I didn't know about them. I'll see if they are easier to level up with. :smiley:

**Reply by Sanglorian** [reply]
Thanks @JaskRendix for this work and @ultidonki for testing it out. Interested to see what your testing finds about the most enjoyable systems, @ultidonki .

**Reply by ultidonki** [reply]
> Interested to see what your testing finds about the most enjoyable systems, @ultidonki .

I've just finished Scoop HQ on my new playthrough, so I was going to wait until the end before posting more thoughts on game balance - I was also thinking, maybe I won't post anything about game balance, only the bugs, and let other people discuss the game balance! Maybe, a lot of people playing and giving their opinions is more important than just mine, other people might enjoy what I didn't, or they might use different tuxemon or different strategies that change how fun or difficult they found the game :smiley: 

However, seeing as you asked, I will post a few things I've found so far:
- Overall I'm finding it a lot more fun, maybe because the experience seems to be better balanced after Route 4. There was very little grinding for levels, in fact most trainers I could beat by healing and going back to fight them again - or going to a different area like Route A or the Dojo and levelling up by beating trainers there. 
The game balance seems quite good for the tuxemon I used - there wasn't a single Tuxemon that could win every battle, so eventually my strongest tuxemon would get defeated by a type match-up, and I'd need to rely on my other 5 tuxemon. This forced me to keep all tuxemon at approximately the same level, as I relied on them so much.
- Experience: I've only been using the default method, and it does seem to give better experience after the Wayfarer Inn/Route 4, as the tuxemon I battle are higher than level 15 or 20-ish. Because the default method uses the square of the level, I have a feeling that the early game might be too hard, and as my tuxemon reach level 40 or higher, it might start getting too easy - but I'll complete the game before I make a judgement about that. 
- I can't remember where exactly where the problems were, but sometimes the difficulty felt uneven - as I've said before, up to Route 4 the difficulty was quite hard if you don't know what you're doing (but now there are nurses on the most difficult routes, so they might be much easier now?) 
From Route A/the Dojo/Nimrod HQ, onto Route 5 and Scoop HQ, there were a few difficulty spikes, and a few places where the tuxemon felt a bit easy/underlevelled compared to me. (I reported one where the first tuxemon in Nimrod HQ had a level 35 tuxemon, after the tuxemon in Route 4 being about 10-15, I think?) Like the first guy you battle in the Mansion was quite a big difficulty jump (level 22 or something, the average tuxemon in the grass in Route 4 was about level 14 I think? And the level 14 tuxemon in the grass always managed to defeat at least 1 tuxemon in my team each time). Another time in Route A, Billie appears near the top, after I'd lost most of my HP/tuxemon from the other trainers - I was like 'Noooo', but maybe I should've been using healing items more? (Actually I have another point about healing items I'll post later) 
Basically, there's a couple of trainers where you just think 'What's up with that level?' I haven't been making a list so it'll have to be someone else who plays through and thinks about this, though.
- Sometimes a tuxemon would use a move and it'd be really powerful and take down one of my far stronger tuxemon. But overall I haven't faced anything too spammy/overpowered yet, so the balance of tuxemon and techniques I've seen so far must be fine I guess.
- The spyder byte caught me by surprise, I wasn't expecting to be stuck with my tuxemon from Route 3 for the next few villages, but it was definitely an experience I guess. My team ended up being surprisingly balanced so far, although I would've preferred cooler-looking tuxemon I guess :smile: I'm interested to hear what other people think of it when Tuxemon gets more players. :smiley: 
- Every so often there's a route with slightly boring/repetitive tuxemon on it, a wider variety might be nice - for example, Route A at night would give you more than a 50% chance to see a katapill, so I thought there was a bug where only katapill were appearing :smile: I got seriously bored of seeing them. I feel like I see a lot of Aardorn/Aardart on most Routes, too. Again I haven't been keeping a list so other people will need to keep an eye out if they play it too.
- I might create an issue for this one, but more item variety would be nice after Route 4 - a Revive and Potion will only recover 20HP, and a Super Potion only recovers 50HP. I think tuxemon have about 100 HP after Route 4, as levels reach 15-20? So it's really difficult/pointless to use healing items in battle. (Healing moves are OK to use though)
- I seem to have a lot of money, but that's probably because I never spend it because potions and super potions are kind of useless - revives are too, because if I revive a tuxemon in battle it will only last 1 turn with 20 HP :smiley: (But that's NOT a complaint, having not enough money sucked at the beginning, of course.)
- Every so often there's just something a bit weird with the dialogue or story, for example - In Route A, Billie says 'I just bought this tuxemon from the Cathedral'! However, she uses the exact same tuxemon she used in the previous route, Route 4. There wasn't a new one. I didn't even note these things down, but that was the most obvious one I can remember. Just every so often I'll see some dialogue and think "Do I know who these people are yet?" or "Have I missed part of the story?" or "Doesn't this contradict something I saw earlier?" Again, I didn't keep a list but something to watch out for if people play again. Overall the story and dialogue made sense, it's just a few tiny places where I noticed something felt 'off' but didn't think it was important enough to write it down - but just wanted to say something in case other people notice things too.

Whew, sorry I didn't take notes for any of the above, so I'm just trying to write stuff based on my experiences I've played recently, I don't remember everything so that was probably very jumbled opinions and stuff that might not be useful to hear. :smiling_face_with_tear: But overall: It's fun! :smile:

**Reply by JaskRendix** [reply]
@ultidonki so, just to clarify, what exactly are you looking for here? Are you asking for more background info on Billie, or is there something else specific you're curious about? From what I can think of, Billie and maybe Dante from the shop are the only two NPCs who really interact with the player throughout the adventure, so I'm not sure what else might be relevant.

> I seem to have a lot of money, but that's probably because I never spend it because potions and super potions are kind of useless - revives are too, because if I revive a tuxemon in battle it will only last 1 turn with 20 HP

I'm gonna say it - I think the auto-healing when you faint is the biggest exploit in the game. I mean, it's way too easy to just respawn and keep going without having to deal with the consequences. But, let's be real, if we were to change it and make players actually have to visit the clinic, I'm sure there would be people crying about it. Some folks just aren't gonna be happy unless they can get away with a free pass.

**Reply by ultidonki** [reply]
> @ultidonki so, just to clarify, what exactly are you looking for here? Are you asking for more background info on Billie, or is there something else specific you're curious about? From what I can think of, Billie and maybe Dante from the shop are the only two NPCs who really interact with the player throughout the adventure, so I'm not sure what else might be relevant.

It was just a general observation, I don't need anything to change right away. But if I played the game from the beginning again, I think I'd notice a few more strange things. If I play through it again, I'll try to write down anything weird I find. 

The biggest problem was when I fought Billie on Route A, she said "I bought this new tuxemon from the Cathedral!" but her tuxemon were the same as last time, there wasn't a new tuxemon. There's just small inconsistencies sometimes if you look for them.

But now that you mention it, some other things aren't explained as much as I expected they would be:
- It's not explained why you need to walk through Nimrod HQ fighting their prototypes, or why you did it (maybe that's the point?)
- It's not explained why you need to walk through Scoop HQ, and it's not totally explained who the guys in blue hats are (mmm... not Omnichannel or Nimrod, but someone different, right? I can't remember exactly who they are. And it's not explained, other than they fight you when they see you.) 
- The spyder byte virus isn't completely explained - tuxemon just start sneezing in the Wayfarer Inn, then you can't put tuxemon in the box because you're told they have a virus. I'm not sure it's mentioned much again until Greenwash HQ etc. I'd already read about the Spyder Byte virus from the wiki so I understood what was happening, but it wasn't explained much. After the Wayfarer Inn, you just go to Route 4, the next village, then Nimrod HQ etc. without much of an explanation.

The above are different from what I was trying to explain above, just another thing I noticed. But they're not very important, I just wanted to point out things I noticed while I was playing. 

> I seem to have a lot of money, but that's probably because I never spend it because potions and super potions are kind of > I'm gonna say it - I think the auto-healing when you faint is the biggest exploit in the game. I mean, it's way too easy to just respawn and keep going without having to deal with the consequences. But, let's be real, if we were to change it and make players actually have to visit the clinic, I'm sure there would be people crying about it. Some folks just aren't gonna be happy unless they can get away with a free pass.

I agree but at the moment, without more healing items of your own, that's the best option. And I think some people just want to play the game and catch tuxemon, so it's nice to have a 'cheaty' way to complete the game without worrying too much. Players who are defeated are still teleported back to the latest healer, so they still get punished a little bit.

**Reply by Sanglorian** [reply]
Thank you, this is really useful feedback. I don't have time just at the moment to action it, but I hope to get a chance to reflect on it and make some changes to the plot and script and so on to make it make more sense.

**Reply by ultidonki** [reply]
> I don't have time just at the moment to action it, but I hope to get a chance to reflect on it and make some changes to the plot and script and so on to make it make more sense.

No problem - and remember it's only my opinion! I'm hoping that after a few more bugs are fixed and the next release is created, I'm hoping more people will play and give their feedback. I'm hoping that'll make it easier to notice problems with game balance etc., as more people can try more things. The story may also be just fine the way it is, maybe other people will understand it better while I'm just being dense, for example :smile: 

And just to make it clear, what I wrote above is just very small considerations to make, I don't think they're big problems at all, the game is perfectly understandable and playable as it is now, I think - these are just very small examples that make me think 'Huh?' for 5 seconds while playing, but they don't stop the game being enjoyable or playable at all, unlike earlier when certain monsters felt over-powered for example :smiley:

**Reply by Sanglorian** [reply]
> I might create an issue for this one, but more item variety would be nice after Route 4 - a Revive and Potion will only recover 20HP, and a Super Potion only recovers 50HP. I think tuxemon have about 100 HP after Route 4, as levels reach 15-20? So it's really difficult/pointless to use healing items in battle. (Healing moves are OK to use though)

Just wanted to say you were totally right about this one! I've got a PR in the works that increases potion healing to 50 and super potion healing to 100.

**Comment by HippasusTwo** [comment]
I added the 'swap ruby for spike' change to my balance branch. I think maybe someone should go through in general and check for where weaker moves are learned later because that doesn't make a great deal of sense: usually you want your best moves to be learned last, although this would be a bit tedious so maybe it could be automated in some way.

edit:

I'll say the thing I found most tedious (besides the horrible healspam and difficulties catching literal caterpillars) was how all the early game moves have terrible accuracy. In Pokemon, one could expect all the early game moves to be 90-100 percent accurate except for status moves. Here it's the opposite: status moves seem super accurate but basic fighting moves not so much. Low accuracy should be for high risk, high reward situations, otherwise it's just frustrating and there is never any reason to keep a less powerful move over a more powerful one if the more powerful is also more accurate.

As to the run away issue, I think maybe there should be an item that lets you run away available at the first store, or some early techniques that let you flee from battle - this can add some catch 'em all difficulty if the monster you want has one of these.

**Reply by Sanglorian** [reply]
> I added the 'swap ruby for spike' change to my balance branch. I think maybe someone should go through in general and check for where weaker moves are learned later because that doesn't make a great deal of sense: usually you want your best moves to be learned last, although this would be a bit tedious so maybe it could be automated in some way.
> 

It's worth mentioning that because we use a refresh mechanic, a "weaker" move may still be desirable because it refreshes faster.

That said, definitely worth tweaking this stuff as people play through the game.

I'm also very happy to increase the accuracy of moves - let us know what you think needs to change and I can at least fix that on the wiki.

**Comment by JaskRendix** [comment]


**Reply by Sanglorian** [reply]
I like brown personally! Thanks for showing all the options.
